.github/workflows/test.yml

```
@@ -53,7 +53,8 @@ jobs:
         terraform:
           - '0.12.30'
           - '0.13.6'
```

Ready for new versions 👍 

```suggestion
          - '0.12.31'
          - '0.13.7'
```